tensorboard/components/tf_paginated_view/tf-paginated-view2.html

```diff
@@ -0,0 +1,414 @@
+<!--
+@license
+Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<link rel="import" href="../paper-button/paper-button.html">
+<link rel="import" href="../paper-input/paper-input.html">
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../tf-dashboard-common/array-update-helper.html">
+<link rel="import" href="../tf-dashboard-common/tensorboard-color.html">
+<link rel="import" href="tf-paginated-view-store.html">
+
+<!--
+  tf-paginated-view2 takes an input list and outputs the input broken
```

Comment:
This documentation is all stale - could you update it?

Also, optional, but given that we want this to fully replace tf-paginated-view, it might be better to avoid the `2` suffix so that it won't linger on years from now even after tf-paginated-view is gone entirely. Perhaps `tf-pagination` or `tf-paginate` or `tf-paginated-repeat`.